IdentityCore/tests/MSIDPkeyAuthHelperTests.m

```diff
@@ -108,6 +108,29 @@ - (void)testCreateDeviceAuthResponse_whenDeviceIsWPJAndAuthServerUrlWihtQueryPar
     }
 }
 
+- (void)testCreateDeviceAuthResponse_whenDeviceIsWPJAndAuthServerUrlWihtQueryParams_andCertAuthoritiesURLEncoded_shouldCreateProperResponse
+{
+    if (@available(iOS 10.0, *))
```

Comment:
What is the minimum version this common objc library supports

Reply:
10, although we should be able to go to 11 soon